osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -21,6 +28,16 @@ private void load(Storage storage, OsuConfigManager config)
                 Bindable = config.GetBindable<ReleaseStream>(OsuSetting.ReleaseStream),
             });
 
+            if (game != null && updateManager != null)
+            {
+                Add(new SettingsButton
+                {
+                    Text = "Check for updates",
+                    Action = updateManager.CheckForUpdate,
+                    Enabled = { Value = game.IsDeployedBuild }
+                });
+            }
```

Comment:
You could just as well do

```suggestion
            Add(new SettingsButton
            {
                Text = "Check for updates",
                Action = updateManager?.CheckForUpdate,
                Enabled = { Value = game?.IsDeployedBuild ?? false }
            });
```

to eliminate the `if` null-checks.